### PR TITLE
[BB-3314] Fix koa integration tests

### PR DESCRIPTION
**Pull request by shimulch**
This PR fixes koa integration tests.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-3314

~~**Discussions**: ~~

**Dependencies**: https://github.com/open-craft/configuration/pull/146

~~**Screenshots**: ~~

~~**Sandbox URL**:~~

**Merge deadline**: None

~~**Testing instructions**:~~

~~**Author notes and concerns**:~~

**Reviewers**
- [ ] @0x29a
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Review by 0x29a — CHANGES_REQUESTED** [review]
Thanks for work on this, @shimulch.

I see that you targeted this PR against `koa-integration-tests`. That's my old branch which I used during planning to understand the scope of required works. You need to change target to master, since almost all changes from this branch are already there :slightly_smiling_face:

File: instance/tests/integration/factories/instance.py [hunk 1]
```diff
@@ -81,6 +81,6 @@ def create(cls, *args, **kwargs):  # pylint: disable=arguments-differ
     # corresponding openedx_release versions from upstream, but can contain custom modifications.
     openedx_release = 'open-release/koa.master'
     configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
-    configuration_version = 'integration-koa'
+    configuration_version = 'shimulch/bb-3314'
     edx_platform_repository_url = 'https://github.com/open-craft/edx-platform.git'
-    edx_platform_commit = 'opencraft-release/koa.master'
+    edx_platform_commit = '0x29a/open-release/koa.master'
```
[the remark this answers is not in the document]
Comment (shimulch) [reply instance/tests/integration/factories/instance.py]: @0x29a, Branch ``opencraft-release/koa.1`` created in ``open-craft/edx-platform``. I've changed references to point to that branch instead.
Also ``0x29a/open-release/koa.master`` has been deleted.

**Comment by shimulch** [comment]
@0x29a I've changed target to ``master`` now. But there seems some commit's not present in the master which are not part of this PR.

**Comment by 0x29a** [comment]
>But there seems some commit's not present in the master which are not part of this PR

@shimulch, changes from following commits already in master:
- https://github.com/open-craft/opencraft/pull/682/commits/3dd5f255da40ecfc5a8478b7fdbb774b69e043be
- https://github.com/open-craft/opencraft/pull/682/commits/31f74c7ab099e5d723047452d60a1271ce14e120
- https://github.com/open-craft/opencraft/pull/682/commits/64c1bb64092b423a48fbeba47d6f319bc0f2f727

I believe you can drop all my commits from this branch, and rebase the last 4 commits on top of master.

**Comment by shimulch** [comment]
@0x29a, Rebased, and quashed with only my changes.

**Pull request merged**